src/components/Filters/Filters.stories.tsx

```diff
@@ -22,6 +22,7 @@ export default {
   component: Filters,
   title: "Components/Filters",
   decorators: [withDimensions(), withRouter(), withBeamDecorator],
+  parameters: { layout: "fullscreen" },
```

Comment:
Prevents the `padding: 1rem;` from being placed on the story's `<body />` tag. I thought about removing it everywhere, but I think it is better to only remove it in stories where the padding causes issues.